src/Components/SmartComponents/SystemsPage/SystemsTableToolbar.js

```diff
@@ -43,12 +51,12 @@ const SystemsTableToolbar = ({
         '',
         {
             label: intl.formatMessage(messages.systemKebabExcludeAnalysis, { count: totalSelectedRows }),
-            onClick: () => doOptOut(null, true),
+            onClick: () => doOptOut(null, selectedRowsData.length === 1 ? selectedRowsData[0].display_name : null, true),
```

Comment:
you have the `totalSelectedRows`

Reply:
Using both `selectedRowsData.length` and `totalSelectedRows` creates a problem of 2 sources of truth :(